app/views/operations/index.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="header">
+  <%= link_to categories_path, class: 'nav-link' do %>
+    <i class="fa-solid fa-arrow-left"></i>
+  <% end %>
+  <h2>Operations</h2>
+  <i class="fa-solid fa-magnifying-glass"></i>
+</div>
+<div class="gray-background">
+  <div class="total-container">
+    <p>Total: <%= number_to_currency(calculate_total(@operations)) %></p>
+  </div>
+  <div class="transactions-container">
+  <% @operations.each do |operation|%>
+    <div class="transaction-container">
+      <div class="transaction-data">
+        <p>Item name: <%= operation.name %></p>
+      </div>
+      <hr class="line">
+      <div class="transaction-data">
+        <p>Item price: <%= number_to_currency(operation.amount) %></p>
+      </div>
+    </div>
```

Comment:
- [ ] Please modify the transactions cards on the transactions page so that it follows closely the Behance design attached below 👇 You may need to attach images, consider typography (font-family, font-weight) and layout of the transaction cards.

### Behance Design
![image](https://github.com/DavidEspinoG/microverse_rails_capstone/assets/46839548/a58028c0-fb49-45ea-ae5f-c9409d008b27)

### Your Design
![image](https://github.com/DavidEspinoG/microverse_rails_capstone/assets/46839548/6f610bf6-a611-4da0-bb5b-55f3311d1138)